tests/test-common.js

```diff
@@ -4,7 +4,7 @@
 /* eslint-disable indent */
 module.exports = async function(options) {
 
-'use strict';
+  'use strict';
```

Comment:
Why this change? It seems that there are a bunch of indentation changes in this PR that are not relevant. Are those mistakes? They seem to ignore the `eslint-disable indent` warning above. Can we back those out if they were not intentional?

Reply:
Ditto.  Hard to review with the indent changes.  In the future make that it's own commit. 
 Alternatively, if we want to have proper indents here, then remove the above eslint pragma.

Reply:
Review-wise it's easier, you can just use the https://github.com/digitalbazaar/jsonld-signatures/pull/90/files?w=1 / `?w=1` option. But yeah, let me see if I can reformat.

Reply:
Out of curiosity, why are the tests wrapped in a `module.exports`?

Reply:
@dmitrizagidulin there is an options parameter so multiple option can be passed in. I actually think it's good design as we could start working on a json manifest file that setups test cases like the `json-ld` ones only in this case for different suites and keys etc.

Reply:
@aljones15 But that's not what it's doing. It's just invoking the whole thing, once. (And iterating between suites /within/ that.)

Reply:
> @aljones15 But that's not what it's doing. It's just invoking the whole thing, once. (And iterating between suites /within/ that.)

I didn't write it, but yeah it is an enigma in our testing conventions. Regardless it does allow you to add new suites with out necessarily writing new tests etc. Perhaps the original author can defend it better.

Reply:
The test wrapping was historically done in jsonld.js due to needing to pass in different apis for node vs browser testing.  It got copied here.  The main entry point test.js and test-karma.js files could pass in what was needed for the platform.  There may be better modern approaches if anyone wants to research that.

Reply:
> The test wrapping was historically done in jsonld.js due to needing to pass in different apis for node vs browser testing. It got copied here. The main entry point test.js and test-karma.js files could pass in what was needed for the platform. There may be better modern approaches if anyone wants to research that.

we actually might considering using it as it generates semantically meanful tests from json which is nice to have. I just wrote 2 test suites in the last 2 weeks and both iterate through 2 different `Suites` so it would have been better to just have a set test suite for each suite, pass in the necessary parameters and go.